components/javascript/src/main/java/org/datacleaner/beans/script/JavaScriptTransformer.java

```diff
@@ -52,9 +52,7 @@
 	private static final Logger logger = LoggerFactory
 			.getLogger(JavaScriptTransformer.class);
 
-	public static enum ReturnType {
-		STRING, NUMBER, BOOLEAN;
-	}
+	public enum ReturnType {STRING, NUMBER, BOOLEAN}
```

Comment:
Honestly I don't care - it goes beyond my own needs for sanitizing code styling, I'm only wondering if anyone else would notice it and frown upon the unstandardized way of doing it :-D